persistence/src/main/resources/migration-scripts/0.4.0-0.5.0.js

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {db} = require("@arangodb");
+const udfs = require("@arangodb/aql/functions");
+const graph = require('@arangodb/general-graph');
+
+console.log("Unregister UDFs");
+
+udfs.unregisterGroup("SPLINE");
+
+console.log("Register SPLINE::OBSERVED_WRITES_BY_READ");
+
+udfs.register(
+    "SPLINE::OBSERVED_WRITES_BY_READ",
+    "/*\n" +
```

Comment:
1. Yeah, we can remove redundant license headers. The only reason why they are there is that the way how I created that string - copy/paste from the corresponding JS file into "|". I just copied the entire original JS file including (obviously) the license header.
Basically, this is what I propose to do for the ease of use and leaving less room for accidental copy mistakes. 

2. I tried multiline string, but it didn't work well with copying/pasting. It required manual treating placeholders like `${blah}` that otherwise interact with the outer back-ticks. IDEA, for instance, doesn't do it correctly on pasting. However for normal strings, auto-escaping works flawlessly. Hence I opted for that.

3. The idea is that you don't modify the UDF code in quotes - you only replace it completely in one chunk. The actual UDF source code remains in the separate JS files. That is where they ought to be maintained. Migration scripts on the other hand are supposed to be _written in stone_. They are written once and forever (excepts probably for snapshot versions that are still in development). Any change to a migration script for an older version defeats the purpose of incremental migration. That's why I intentionally didn't use any dynamic content inclusion mechanism or such - only static content.